src/index.js

```diff
@@ -0,0 +1,74 @@
+import './style.css';
+
+import { tasks, add } from './modules/add.js';
+
+import remove from './modules/remove.js';
+
+import status from './modules/status.js';
+
+import {
+  edit, outclick,
+} from './modules/edit.js';
+
+function removecompleted() {
+  const completedarray = tasks.filter((tasks) => tasks.completed === true);
+  for (let i = 0; i < completedarray.length; i += 1) {
+    const index = (completedarray[i].index - 1) - i;
+    tasks.splice(index, 1);
+  }
+  for (let i = 0; i < tasks.length; i += 1) {
+    tasks[i].index = i + 1;
+  }
+  localStorage.setItem('array', JSON.stringify(tasks));
+}
```

Comment:
I believe you've done a great job implementing the removecompleted function. However, I think using a forEach loop inside the function would have been more efficient than using a regular for loop.